components/pages/_pages.css

```diff
@@ -0,0 +1,9 @@
+/*
+  Pages
```

Comment:
I've never had pages section in my style guide. Do you have an example somewhere how this section may look like? 🙂

Reply:
The point of this section is to allow us to have samples of fully rendered pages, rather than just design components. Pages I imagine we'd show here would be those from the designs
- home
- basic
- article
- recipe
- etc

(not just the node/view mode, but fully rendered with header/footer/sidebar/etc as well)

Reply:
Is the idea to use HTML or just images? If we will do it using HTML, this sounds pretty ambitious but I'm open to give it a go 🙂

Reply:
I'm not against it neither, but I'd leave it as low priority.

Reply:
The point of this pull request is that it converts _most_ of the components to kss style components using twig instead of php.

Let's have the discussion about the /pages/ function later or in a separate thread. I added it initially because that's what patternlab does - create atoms > molecules > organisms > templates > pages

It's easy for us to just remove this directory, but for now can we get what is done merged, so others can contribute from that commit.

Reply:
Here's an example of a fully rendered component page in patternlab:

Header - language switcher + branding + menu atoms = organism
Main - homepage node (made up of other atoms/organisms)
Total = Header + Main = Page

![screen shot 2017-10-07 at 11 29 35](https://user-images.githubusercontent.com/2183332/31306962-dc1d1c76-ab52-11e7-8489-73ee53d57bca.png)